test-clients/src/main/java/com/hedera/services/bdd/suites/perf/crypto/CryptoTransferPerfSuiteWOpProvider.java

```diff
@@ -0,0 +1,130 @@
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.BUSY;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.DUPLICATE_TRANSACTION;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INSUFFICIENT_ACCOUNT_BALANCE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INSUFFICIENT_PAYER_BALANCE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SIGNATURE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PAYER_ACCOUNT_NOT_FOUND;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PLATFORM_TRANSACTION_NOT_CREATED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_EXPIRED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNKNOWN;
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+
+public class CryptoTransferPerfSuiteWOpProvider extends HapiApiSuite {
+	private static final Logger log = LogManager.getLogger(CryptoTransferPerfSuiteWOpProvider.class);
+	public static void main(String... args) {
+		CryptoTransferPerfSuiteWOpProvider suite = new CryptoTransferPerfSuiteWOpProvider();
+		suite.setReportStats(true);
+		suite.runSuiteSync();
+	}
+
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(runMixedTransferAndSubmits());
+	}
+
+
+	@Override
+	public boolean hasInterestingStats() {
+		return true;
+	}
+
+
+	private HapiApiSpec runMixedTransferAndSubmits() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+		return defaultHapiSpec("CryptoTransferPerfSuiteWOpProvider")
+				.given(
+						withOpContext((spec, ignore) -> settings.setFrom(spec.setup().ciPropertiesMap())),
+						logIt(ignore -> settings.toString())
+				).when().then(
+						runWithProvider(XfersFactory())
+								.lasting(settings::getMins, () -> MINUTES)
+								.maxOpsPerSec(settings::getTps)
+				);
+	}
+
+
+	private Function<HapiApiSpec, OpProvider> XfersFactory(){
+		return spec -> new OpProvider() {
+			@Override
+			public List<HapiSpecOperation> suggestedInitializers() {
+				return List.of(
+						cryptoCreate("sender")
+								.balance(10000000000L)
+								.hasRetryPrecheckFrom(BUSY, PLATFORM_TRANSACTION_NOT_CREATED),
+						cryptoCreate("receiver")
+								.hasRetryPrecheckFrom(BUSY, PLATFORM_TRANSACTION_NOT_CREATED),
+						sleepFor(10_000L));
+			}
+
+
+			@Override
+			public Optional<HapiSpecOperation> get() {
+				final var op = cryptoTransfer(tinyBarsFromTo("sender", "receiver", 1L))
+						.noLogging()
+						.hasKnownStatusFrom(SUCCESS, OK, INSUFFICIENT_PAYER_BALANCE
+								, UNKNOWN, TRANSACTION_EXPIRED,
+								INSUFFICIENT_ACCOUNT_BALANCE)
+						.hasRetryPrecheckFrom(BUSY, DUPLICATE_TRANSACTION,
```

Comment:
It's probably better to add `.hasPrecheckFrom(DUPLICATE_TRANSACTION, OK)` and remove `OK` from `.hasKnownStatusFrom(...)` and `DUPLICATE_TRANSACTION` from `.hasRetryPrecheckFrom(...)`.

Reply:
Fixed !